### DISCUSSION TITLE
add table styles

### DISCUSSION THREAD

**Discussion by K3yr0nym0us** [TomSchimansky/CustomTkinter#1255] [Ideas]
I love this library it makes designing apps with tkinter a lot easier but I am disappointed that there is no table widget.
It would be a great improvement since it is one of the things that is used the most when programming applications.

**Comment by Akascape** [comment]
@K3yr0nym0us Here is a table widget for customtkinter: https://github.com/Akascape/CTkTable
![Screenshot](https://user-images.githubusercontent.com/89206401/233420929-bf210cb3-5b5f-49b2-ba7a-f01d187e72cf.jpg)

**Comment by K3yr0nym0us** [comment]
Beautiful, good way to create.
10/10
